source/getting-started/how-to-create-a-struts2-web-application.md

```diff
@@ -176,6 +176,52 @@ Using both `log4j-core` and `log4j-api` allows to use [the latest version of Log
 
 Note the above log4j2 configuration specifies the console as the log target.
 
+Optionally, if using maven bom "bill of materials" in `dependencyManagement` section for both Struts and log4j2,
+`pom.xml` will look like. Note that this way you can omit `version` line for every used module
+
+```xml
+<properties>
+	<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+	<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+	<java.version>1.8</java.version>
+
+	<struts2.version>2.5.14</struts2.version>
+	<log4j2.version>2.10.0</log4j2.version>
+</properties>
+
+<dependencyManagement>
+	<dependencies>
+		<dependency>
+			<groupId>org.apache.struts</groupId>
+			<artifactId>struts2-bom</artifactId>
+			<version>${struts2.version}</version>
+			<type>pom</type>
+			<scope>import</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.logging.log4j</groupId>
+			<artifactId>log4j-bom</artifactId>
+			<version>${log4j2.version}</version>
+			<scope>import</scope>
+			<type>pom</type>
+		</dependency>
+	</dependencies>
+</dependencyManagement>
+
+<dependencies>
+	<!-- Struts 2 -->
+	<dependency>
+		<groupId>org.apache.struts</groupId>
+		<artifactId>struts2-core</artifactId>
+	</dependency>
+
+	<dependency>
+		<groupId>org.apache.logging.log4j</groupId>
+		<artifactId>log4j-core</artifactId>
```

Comment:
I would add `log4j-api` as well

Reply:
log4j-api is already managed in second bom import.  
So no need to specify it just to put correct version.

I just tried it today, no need.

Reply:
There is other problem as the example will have red problem in Eclipse IDE for `index.jsp`,  until servlet dependency is added
```xml
		<dependency>
			<groupId>javax.servlet</groupId>
			<artifactId>servlet-api</artifactId>
			<version>2.4</version>
			<scope>provided</scope>
		</dependency>
```

Reply:
![image](https://user-images.githubusercontent.com/1614482/33604055-954781f8-d9ef-11e7-87ca-9792d9bfb175.png)

Reply:
Hm... up to you if the `servlet-api` should be defined in the `pom.xml`

Reply:
servlet-api would be out of this PR